src/App.tsx

```
@@ -374,10 +835,24 @@ const App: Component = () => {
               <span class="material-symbols-outlined text-[14px]">content_copy</span>
               <span>Copy</span>
             </button>
+            <div class="w-px h-5 bg-slate-200"></div>
+            <button
+              onClick={() => appStore.setShowDebugPanel(!appStore.showDebugPanel())}
+              class={`flex items-center gap-1.5 px-3 py-1.5 text-[10px] font-bold rounded-lg transition-all shadow-md active:scale-95 active:shadow-sm ${appStore.showDebugPanel() ? 'bg-slate-700 text-white hover:bg-slate-800' : 'bg-slate-200 text-slate-600 hover:bg-slate-300'}`}
+              title={appStore.showDebugPanel() ? 'Hide debug panel (improves performance)' : 'Show debug panel'}
+            >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)